test/spec.js

```diff
@@ -280,4 +281,35 @@ describe('L.Sync', function () {
             a._syncMaps.should.eql([]);
         });
     });
+    
+    describe('sync with circle marker', function () {
```

Comment:
Unfortunatly I'm not (yet) a Moccha user, it's just a non-regression testing to ensure that synced maps still work. I'm sure a community user could write better tests to check markers position.